### DISCUSSION TITLE
blocks color coded (using ratings) + timer calculator

### DISCUSSION THREAD

**Discussion by Cybeuu** [earcherc/deep-work#9] [Ideas]
![image](https://github.com/user-attachments/assets/c12767af-d5c1-4974-9f5e-8d4090aa2c0f)

- deep work rating = color depth of deep work block 
- and user can write short bullet comments about their deep work sesh. 
- short and bullet to make it compact but also to force user to think of that one thing resulting in low/high rating rather than a whole story. 

- also, a simple time lapse could also be placed beside the deep work time log (5 h 30min as shown in pic above)

**Comment by earcherc** [comment]
Awesome! Will get in this weekend
